docs/webui.md

```diff
@@ -0,0 +1,206 @@
+
+The main content displays a created metalake list.
+
+### Oauth mode
+
+```
+gravitino.authenticator = oauth
+```
+
+If the configuration parameter for `gravitino.authenticator` is set to `oauth`, the webUI will directly display the login page.
+
+![webui-login-with-oauth](assets/webui/login-with-oauth.png)
+
+1. Follow the [example steps](security#example), enter the following values in these text fields.
+
+- Client ID: `test`
+- Client Secret: `test`
+- Scope: `test`
+
+2. Clicking on the `LOGIN` button to verify redirection to the homepage.
+
+![webui-metalakes-oauth](assets/webui/metalakes-oauth.png)
+
+At the top-right, there is an extra icon button that redirects to the login page when clicked.
+
+## Manage metadata
+
+> All the manage actions are from [REST API](api/rest/gravitino-rest-api)
+
+### Metalake
+
+#### [Create a metalake](getting-started#using-rest-to-interact-with-gravitino)
+
+On the homepage, clicking the `CREATE METALAKE` button will display a dialog to create a metalake.
+
+![create-metalake-dialog](assets/webui/create-metalake-dialog.png)
+
+Create a metalake needs these fields:
+
+1. **Name**(**_required_**): the metalake name, It's best to use only English letters, and avoid using special characters like _space_.
+2. **Comment**(_optional_): the comment of this metalake
+3. **Properties**(_optional_): clicking on the `ADD PROPERTY` button to add custom properties
+
+![metalake-list](assets/webui/metalake-list.png)
+
+Each metalake consists of three actions.
+
+![metalake-actions](assets/webui/metalake-actions.png)
+
+#### 1. Show metalake details
+
+![metalake-details](assets/webui/metalake-details.png)
+
+#### 2. Edit metalake
+
+Displays a dialog for modifying fields.
+
+![create-metalake-dialog](assets/webui/create-metalake-dialog.png)
+
+#### 3. Delete metalake
+
+Display a confirmation dialog, clicking `SUBMIT` will delete this metalake.
+
+![delete-metalake](assets/webui/delete-metalake.png)
+
+### Catalog
+
+Clicking on `test`(metalake_name) on the table to view catalogs in a metake.
+
+In the first time, It shows no data until create a catalog.
+
+![metalake-catalogs](assets/webui/metalake-catalogs.png)
+
+Clicking on the Tab - `DETAILS` to view this metalake details in metalake catalogs page.
+
+![metalake-catalogs-details](assets/webui/metalake-catalogs-details.png)
+
+#### Create a catalog
+
+Clicking on the `CREATE CATALOG` button will displaying a dialog to create a catalog.
+
+![create-catalog](assets/webui/create-catalog.png)
+
+Create a catalog needs these fields:
+
+1. **Catalog name**(**_required_**): the catalog name of this metalke
+2. **Type**(**_required_**): the default value is `relational`,
+3. **Provider**(**_required_**): `hive`/`iceberg`/`mysql`/`postgresql`
+4. **Comment**(_optional_): the comment of this catalog
+5. **Properties**(**specific provider inculdes required fileds**)
+
+##### Providers
+
+> Required properties in various providers
+
+- hive
```

Comment:
Capitalize the first letter for Hive, Iceberg, etc.

Reply:
> Capitalize the first letter for Hive, Iceberg, etc.

DONE